### PR TITLE
Building doc on travis

### PR DESCRIPTION
**Pull request by petrus-v**
this is a purpose about #56

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@ install:
   - pip install coverage
 
 script:
-  - flake8 anybox 
-  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox 
+  - flake8 anybox
+  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox
```
Comment (bwrsandman) [inline .travis.yml]: Just wondering why we can't use the more pythonic:

* `python setup.py flake8`
* `python setup.py tests`

Comment (lmignon) [inline .travis.yml]: @bwrsandman to use the same commands as those specified in the documentation http://docs.anybox.fr/anybox.recipe.odoo/current/contributing.html#launching-static-analysis-and-unit-tests

Comment (bwrsandman) [inline .travis.yml]: `flake8 anybox` doesn't cover as much as `setup.py flake8`

```
$ flake8 anybox
# Nothing
$ python setup.py flake8
running flake8
setup.py:20:14: E225 missing whitespace around operator
setup.py:20:22: E231 missing whitespace after ','
setup.py:53:7: E121 continuation line under-indented for hanging indent
```

We could change the doc

**Comment by petrus-v** [comment]
I close this PR, #59 build the documentation

**Comment by petrus-v** [comment]
I close this PR, #59 build the documentation

**Pull request closed without merging**